tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_container_validation.py

```diff
@@ -221,7 +220,7 @@ def test_valid_containers(
                 Section(kind=SectionKind.TYPE, data="00800000"),
                 Section.Data("da"),
             ],
-            expected_bytecode="ef0001 010004 040001 00 00800000 da",
+            expected_bytecode="ef0001 010004 ff0001 00 00800000 da",
```

Comment:
I think we can remove these instead of changing.

Reply:
hm, this has been proposed in this PR threads. Right now the cost is already sunk, so maybe makes sense to commit to removing these afterwards? Don't we need unit tests for `Container` class first in order to safely remove `expected_bytecode` - @winsvega mentioned this, and I think I agree

Reply:
Yes, sure they can stay for now. I mostly mean it: don't waste time on fixing these next time.